packages/devel/boost/package.mk

```diff
@@ -35,10 +35,10 @@ configure_target() {
                   --with-python=${TOOLCHAIN}/bin/python \
                   --with-python-root=${SYSROOT_PREFIX}/usr
 
-  echo "using gcc : $(${CC} -v 2>&1  | tail -n 1 |awk '{print $3}') : ${CC}  : <compileflags>\"${CFLAGS}\" <linkflags>\"${LDFLAGS}\" ;" \
-    > tools/build/src/user-config.jam
-  echo "using python : ${PKG_PYTHON_VERSION/#python} : ${TOOLCHAIN} : ${SYSROOT_PREFIX}/usr/include : ${SYSROOT_PREFIX}/usr/lib ;" \
-    >> tools/build/src/user-config.jam
+  echo "using gcc : $(${CC} -v 2>&1  | tail -n 1 | awk '{print $3}') : ${CC}  : <compileflags>\"${CFLAGS}\" <linkflags>\"${LDFLAGS}\" ;" \
+    >tools/build/src/user-config.jam
+  echo "using python : ${PKG_PYTHON_VERSION/#python/} : ${TOOLCHAIN} : ${SYSROOT_PREFIX}/usr/include : ${SYSROOT_PREFIX}/usr/lib ;" \
```

Comment:
Does the script cite why it's adding a `/` here? It doesn't match bash's documentation on string manipulation. See https://tldp.org/LDP/abs/html/string-manipulation.html and https://www.gnu.org/software/bash/manual/bash.pdf section 3.5.3.

Reply:
I thinks is correct, bash says `${var/pattern/string}` - just we replace it with nothing so the string part is empty.
BUT after testing we are actually doing wrong? formatted `${var#pattern}`. No idea what effect `/` before `#python`  should do.

```
cvh@PC:/LE$ export PKG_PYTHON_VERSION="python3.11"
echo '${PKG_PYTHON_VERSION/#python/}'"=${PKG_PYTHON_VERSION/#python/}"
echo '${PKG_PYTHON_VERSION/#python}'"=${PKG_PYTHON_VERSION/#python}"
echo '${PKG_PYTHON_VERSION#python}'"=${PKG_PYTHON_VERSION#python}"

${PKG_PYTHON_VERSION/#python/}=3.11
${PKG_PYTHON_VERSION/#python}=3.11
${PKG_PYTHON_VERSION#python}=3.11
```

So my conclusion is it should be `${PKG_PYTHON_VERSION#python}` ?

Reply:
Looks right.